src/components/StudentInterestSection.tsx

```diff
@@ -0,0 +1,85 @@
+interface StudentInterestSectionProps {
+  studentInterestStyles: React.CSSProperties
+  description: string
+  url: string
+}
+
+export default function StudentInterestSection({
+  studentInterestStyles,
+  description,
+  url,
+}: StudentInterestSectionProps) {
+  return (
+    <>
+      {/* Define CSS styles for hover effect */}
+      <style>
+        {`
+                .details-container-studentInterest {
+                  padding: 0;
+                  background-color: white;
+                  cursor: pointer;
+                  font-size: 18px;
+                  transition: background-color 0.3s ease;
+                }
+      
+                .details-container-studentInterest:hover {
+                  background-color:rgb(230, 249, 224);            
+                }
+      
+                .details-summary {
+                  display: flex;
+                  align-items: center;
+                  font-weight: bold;
+                  color: #282F34;
+                  cursor: pointer;
+                  list-style: none;
+                }
+              `}
+      </style>
+
+      <div style={{ margin: '1% 40% 0% 5%', ...studentInterestStyles }}>
+        <span style={{ color: 'green' }}>#</span>&nbsp;Student Interest Form
```

Comment:
Not sure if that nbsp is doing anything

```suggestion
        <span style={{ color: 'green' }}>#</span>Student Interest Form
```

Reply:
It does. It create a whitspace between # and Student. For some reason normal whitespace character did not work here and stack overflow recommended using &nbsp;